custom-connectors/ServiceNow/Readme.md

```diff
@@ -0,0 +1,80 @@
+
+### NOTE
+> This is a sample connector for ServiceNow to illustrate how to use basic CRUD operations along with added OAuth authentication and attachments support. ServiceNow improves service levels, energizes employees, and enables your enterprise to work at light speed
+
+## ServiceNow [Sample] Connector
+This is a sample connector provided here with the intent to illustrate basic create, read, update & delete operations on ServiceNow Incident records, support for OAuth authentication and attachments
+
+Public connector: https://docs.microsoft.com/en-us/connectors/service-now/
```

Comment:
Make this into bullets so that it shows up on different lines

Reply:
Made this change